examples/fluids/advection.h

```diff
@@ -121,27 +124,36 @@ static inline int Exact_Advection(CeedInt dim, CeedScalar time,
   }
 
   // Initial Conditions
-  q[0] = 1.;
-  q[1] = -(y - center[1]);
-  q[2] =  (x - center[0]);
-  q[3] = 0.0;
-  CeedInt continuityBubble = -1;
-  // 0 is original sphere, switch to -1 to challenge solver with sharp gradients in back half of bubble
-  switch (continuityBubble) {
-  // original continuous, smooth shape
-  case 0: {
-    q[4] = r <= rc ? (1.-r/rc) : 0.;
-  } break;
-  // discontinuous, sharp back half shape
-  case -1: {
-    q[4] = ((r <= rc) && (y<center[1])) ? (1.-r/rc) : 0.;
-  } break;
-  // attempt to define a finite thickness that will get resolved under grid refinement
-  case 2: {
-    q[4] = ((r <= rc)
-            && (y<center[1])) ? (1.-r/rc)*fmin(1.0,(center[1]-y)/1.25) : 0.;
-  } break;
-  }
+  switch (context->wind_type) {
+  case 0:
+    q[0] = 1.;
+    q[1] = -(y - center[1]);
+    q[2] =  (x - center[0]);
+    q[3] = 0.;
+    CeedInt continuityBubble = -1;
+    // 0 is original sphere, switch to -1 to challenge solver with sharp gradients in back half of bubble
+    switch (continuityBubble) {
+    // original continuous, smooth shape
+    case 0: {
+      q[4] = r <= rc ? (1.-r/rc) : 0.;
+    } break;
+    // discontinuous, sharp back half shape
+    case -1: {
+      q[4] = ((r <= rc) && (y<center[1])) ? (1.-r/rc) : 0.;
+    } break;
+    // attempt to define a finite thickness that will get resolved under grid refinement
+    case 2: {
+      q[4] = ((r <= rc)
+              && (y<center[1])) ? (1.-r/rc)*fmin(1.0,(center[1]-y)/1.25) : 0.;
+    } break;
+    }
+    case 1:
+      q[0] = 1.;
+      q[1] = wind[0];
+      q[2] = wind[1];
+      q[3] = wind[2];
+      q[4] = 0.;
```

Comment:
@jedbrown , is this what you had in mind or do we need a bubble in the domain?

Reply:
How about the same blobs as for rotation?

Reply:
Yeah. For some reason I thought we would prefer an empty tunnel. In fact, I have found that presence of those blobs helps with spotting bugs and visualization.